### DISCUSSION TITLE
How do i unlink the GitHub Slack Integration just from DMs?

### DISCUSSION THREAD

**Discussion by zenshayan** [community/community#108045] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

The github bot integration on the Slack has a link button in its DM but does not have an unlink slash command. I've searched its settings for an hour with no luck.

How do i stop the GitHub bot from tagging my slack username in places, or better yet, just unlink the bot from my personal account, but still keep it in the slack workspace and all its slack channels.

**Comment by EnnyToluwase** [comment, marked as answer]
You can stop the `@mention` by revoking your authorization of the Slack App

https://docs.github.com/en/apps/using-github-apps/reviewing-and-revoking-authorization-of-github-apps#reviewing-your-authorized-github-apps
